development-defaults.ini

```diff
@@ -1,5 +1,5 @@
 
-path = "/uber"
```

Comment:
yea, in fact this might... already be puppetized?  if so we could leave it as "/rams" here and magfest could override it (if we actually even cared that much, "/rams" might be fine)